README.md

```diff
@@ -518,41 +574,32 @@ document.addEventListener('pjax:send', topbar.show)
 document.addEventListener('pjax:complete', topbar.hide)
 ```
 
-### HTTP Headers
+## HTTP Headers
 
-Pjax uses several custom headers when it makes and receives HTTP requests. 
-If the requests are going to your server, you can use those headers for
-some meta information about the response.
+Pjax uses several custom headers when it makes and receives HTTP requests. If the requests are going to your server, you can use those headers for some meta information about the response.
 
-##### Request headers
+### Request Headers
 
 Pjax sends the following headers with every request:
 
 * `X-Requested-With: "XMLHttpRequest"`
 * `X-PJAX: "true"`
-* `X-PJAX-Selectors`: A serialized JSON array of selectors, taken from 
-`options.selectors`. You can use this to send back only the elements that 
-Pjax will use to switch, instead of sending the whole page. Use `JSON.parse()` 
-server-side to deserialize it back to an array.
+* `X-PJAX-Selectors`
+    * A serialized JSON array of selectors, taken from `options.selectors`. You can use this to send back only the elements that Pjax will use to switch, instead of sending the whole page. Note that you'll need to deserialize this on the server (Such as by using `JSON.parse()`)
 
-##### Response headers
+### Response Headers
 
 Pjax looks for the following headers on the response:
 
 * `X-PJAX-URL` or `X-XHR-Redirected-To`
 
-  Pjax first checks the `responseURL` property on the XHR object to 
-  check if the request was redirected by the server. Most browsers support 
-  this, but not all. To ensure Pjax will be able to tell when the request 
-  is redirected, you can include one of these headers with the response, 
-  set to the final URL.
+Pjax first checks the `responseURL` property on the XHR object to see if the request was redirected by the server. Most browsers support this, but not all. To ensure Pjax will be able to tell when the request is redirected, you can include one of these headers with the response, set to the final URL.
 
-
-#### Note about DOM ready state
+### A Note about DOM Ready State
```

Comment:
IMO the old way looked better.

@robinnorth ?

Reply:
Certainly we should settle on either _Title Case_ or _Sentence case_ throughout, 'about' should be capitalised here if we keep title case.

I usually have a preference for sentence case, FWIW.

Reply:
>I usually have a preference for sentence case, FWIW.

I do as well (unless it's a proper noun). There are a few places where that was changed.

Truthfully, here I was focusing more on the `A`.

Reply:
Ah, I see. I think either is OK, it could also just be shortened to _'DOM ready state'_. I leave it up to you.

Having re-read it in situ, it seems that it should be a level 2 heading so that it does not appear to be a sub-heading of the preceding _HTTP headers_ section.

Reply:
Changed to `## Dom Ready State`